src/elements/content-sidebar/SidebarResponsiveness.scss

```diff
@@ -0,0 +1,154 @@
+@import '../common/variables';
+
+.u-md-bcs-SidebarNavSign-icon {
+    display: none;
+}
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNavSign-icon {
+            display: none;
+        }
+
+        .u-md-bcs-SidebarNavSign-icon {
+            display: inline;
+            width: 20px;
+            height: 20px;
+            margin-top: 2px; //for visual alignment
+        }
+
+        .bcpr {
+            .bcpr-body {
+                position: static;
+                flex-direction: column;
+            }
+
+            .bcpr-container {
+                position: static;
+            }
+        }
+
+        .preview-body {
+            height: 100%;
```

Comment:
Relative height is notoriously difficult to work with, since it has to either have a fixed-height parent or go all the way up to the root of the document. Should we use flexbox or absolute positioning, instead?

Reply:
Instead of using `height: 100%` I've updated it to use `flex: basis: 100%` instead.